services/brig/test/integration/Main.hs

```diff
@@ -80,31 +72,17 @@ runTests iConf bConf = do
         , teamApis
         , turnApi
         ]
+  where
+    mkRequest (Endpoint h p) = host (encodeUtf8 h) . port p
 
 main :: IO ()
 main = withOpenSSL $ do
   (iPath, bPath) <- parseConfigPaths
-  iConf <- join $ handleParseError <$> decodeConfigFile iPath
-  bConf <- join $ handleParseError <$> decodeConfigFile bPath
+  iConf <- join $ handleParseError <$> decodeFileEither iPath
+  bConf <- join $ handleParseError <$> decodeFileEither bPath
 
   runTests iConf bConf
 
-initCassandra :: Opts.Endpoint -> Logger -> IO Cql.ClientState
-initCassandra ep lg =
-    Cql.init lg $ Cql.setPortNumber (fromIntegral $ Opts.port ep)
-                . Cql.setContacts (unpack (Opts.host ep)) []
-                . Cql.setKeyspace (Cql.Keyspace "brig_test")
-                $ Cql.defSettings
-
-mkRequest :: Endpoint -> Request -> Request
-mkRequest (Endpoint h p) = host (B.pack h) . port p
-
-handleParseError :: (Show a) => Either a b -> IO (Maybe b)
-handleParseError (Left err) = do
-  putStrLn $ "Parse failed: " ++ show err ++ "\nFalling back to environment variables"
-  pure Nothing
-handleParseError (Right val) = pure $ Just val
-
 parseConfigPaths :: IO (String, String)
 parseConfigPaths = do
```

Comment:
You're not using the `IsOption` types here introduced in types-common with this PR - why not?